docs/quickstarts/nextjs.mdx

```diff
@@ -277,83 +268,113 @@ export default function Example() {
 
 ### Server side
 
-#### API Routes
+#### App Router
 
-You can protect your API routes by using the [`getAuth`](/docs/references/nextjs/get-auth) helper and retrieve data from your own systems.
+[`auth()`][auth-ref] and [`currentUser()`][currentuser-ref] are new helpers that you can use inside of your Route Handlers, middleware, Server Components, and Server Actions.
 
-```tsx filename="pages/api/auth.ts"
-import type { NextApiRequest, NextApiResponse } from "next";
-import { getAuth } from "@clerk/nextjs/server";
+The [`auth()`][auth-ref] helper will return the [`Authentication`](/docs/references/nextjs/authentication-object) object of the currently active user. Since request data like `headers()` and `cookies()` is now available in the global scope for Next.js, you no longer need to explicitly pass a `Request` object to Clerk.
 
-export default function handler(req: NextApiRequest, res: NextApiResponse) {
-  const { userId } = getAuth(req);
-  if (!userId) {
-    res.status(401).json({ error: "Unauthorized" });
-    return;
-  }
-  // retrieve data from your database
-  res.status(200).json({});
-}
-```
+The [`currentUser()`][currentuser-ref] helper will return the [`User`][user-object] object of the currently active user. This is helpful if you want to render information, like their first and last name, directly from the server.
 
-In some cases, you may need the full [`User`](/docs/references/javascript/user/user) object. For example, if you want to access the user's email address or name. You can use the [`clerkClient`](/docs/references/backend/overview) helper to get the full `User` object.
+<Callout type="info">
+  Any requests from a Client Component to a Route Handler will automatically include the user's current token as the Bearer token.
+</Callout>
 
+<Tabs items={["Route Handler", "Route Handler w/ User Fetch"]}>
+  <Tab>
+    You can protect your API routes by using the [`auth()`](/docs/references/nextjs/auth) helper, if the route is not protected by [middleware](/docs/references/nextjs/auth-middleware). 

```

Comment:
I'm not sure if this statement is fully accurate. As far as I know, `auth()` requires the middleware to have executed. Is there a way we can clarify this?

Reply:
This is correct, actually. I added that too quickly thinking of a reason we are checking userId when we have middleware. I'll clean it up, maybe change the example.

Reply:
@royanger just a gentle reminder to tweak this wording 👀

Reply:
@BRKalow -- 100%. Was waiting on storing out a few other issues then was going to do on larger edit on this.